### ISSUE TITLE
[BUG] Intermittent Loading of Card Images in Add Payment Method UI

### ISSUE THREAD

**Issue by seanirby** [Adyen/adyen-ios#834]
**Describe the bug**
Our React Native app uses `adyen-ios` via the `react-native-adyen-payment` npm package.   When adding a new payment method in the app we are seeing the card types come in intermittently on the form.  They either all show up or none.  To trigger this UI we call [AdyenPayment.startPaymentPromise](https://mkharibalaji.github.io/react-native-adyen-payment/#/LISTENERMETHODS?id=startpaymentpromisecomponentcomponentdatapaymentdetails) .  Here's an example call:

```
  await AdyenPayment.startPaymentPromise(
    'scheme',
    {
      locale: "en_GB", 
      scheme: {
         card_public_key: …
      }, 
      bcmc: {
         card_public_key: …
      }
    },
    {
      reference: "XX"
      shopperReference: "XXXXXXXX"
      shopperEmail: "XX@XXXX.com"
      channel: "IOS"
      returnUrl: "levi://adyen"
      merchantAccount: ""
      additionalData: {}
      amount: {value: 200, currency: "EUR"}
      countryCode: "GB"
      shopperLocale: "en_GB"
    }
  );
```

As far as I can tell there is no change in the data being provided in cases where the logos show and when they don't show.  If you have any ideas on how to root cause this particular issue I would greatly appreciate it.  Thanks

**Screenshots**
![Simulator Screen Shot - iPhone 11 - 2022-03-21 at 09 15 48](https://user-images.githubusercontent.com/2643572/159304059-b06dca32-4bb9-467b-873a-04d7b826d05d.png)
![Simulator Screen Shot - iPhone 11 - 2022-03-21 at 09 15 33](https://user-images.githubusercontent.com/2643572/159304064-534d005c-1452-456c-99f8-88b7506c1ec7.png)

**Smartphone (please complete the following information):**
 - Device: iPhone11
 - OS: iOS 15.4
 - SDK Version 3.3.0

**Comment by descorp** [comment]
Hey @seanirby 

Thanks for reaching out!

This sounds like an old bug we have fixed in v4.1.0.
I am suggest to update Adyen SDK to the latest version.

---

Bad news: Adyen is not affiliated with developers of `react-native-adyen-payment` and we can't directly help with this issue - you have to communicate it with them. You can tag me - best we can do is to help locating necessary changes.

Good news: we are working on [Adyen's official React Native](https://github.com/Adyen/adyen-react-native) library. 
It is still in deep BETA, so any feedback is welcome!

**Comment by descorp** [comment]
Due to lack of activity I am going to close this ticket.
Feel free to continue in this thread
